libsplinter/src/circuit/handlers/direct_message.rs

```diff
@@ -20,12 +20,24 @@ use crate::protos::circuit::{
     CircuitDirectMessage, CircuitError, CircuitError_Error, CircuitMessageType,
 };
 
+#[cfg(feature = "service-message-handler-dispatch")]
+use crate::error::InternalError;
+#[cfg(feature = "service-message-handler-dispatch")]
+use crate::{
+    runtime::service::ServiceDispatcher,
+    service::{
+        CircuitId as Service2CircuitId, FullyQualifiedServiceId, ServiceId as Service2ServiceId,
```

Comment:
Remove the "2" here.

Reply:
Replaced `Service2-` with `Routing-` in both cases.

Reply:
Updated to remove `Service2-` and add `Routing-` to the `circuit::routing::ServiceId` usage (via its own commit).